src/main/java/net/masterthought/cucumber/generators/AbstractPage.java

```diff
@@ -70,7 +70,7 @@ private void generateReport() {
 
         Template template = engine.getTemplate("templates/generators/" + templateFileName);
         File reportFile = new File(configuration.getReportDirectory(),
-                ReportBuilder.BASE_DIRECTORY + File.separatorChar + getWebPage());
+                ReportBuilder.BASE_DIRECTORY + configuration.getDirectoryQualifier() + File.separatorChar + getWebPage());
```

Comment:
and here :)

Reply:
No, it's not missing; it's not needed - as this is just a suffix to the current directory name. It can be passed as _abcd for example.